public/app/features/dashboard/components/SaveDashboard/forms/SaveDashboardForm.tsx

```diff
@@ -124,3 +143,14 @@ export const SaveDashboardForm = ({
     </Form>
   );
 };
+
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    message: css`
+      display: flex;
+      align-items: end;
+      flex-direction: column;
+      width: 100%;
```

Comment:
@ivanortegaalba we have switched to object syntax here a long time ago, @ashharrison90 even updated all existing code to use the better typed object syntax